pelutils/ds/plots.py

```diff
@@ -1,32 +1,25 @@
 from __future__ import annotations
-from typing import Any, Callable, Iterable
+from copy import deepcopy
+from typing import Any, Callable, Iterable, Optional
 
 from . import _import_error
 try:
+    import matplotlib as mpl
     import matplotlib.pyplot as plt
     import matplotlib.colors as mcolour
     from scipy import stats
 except ModuleNotFoundError as e:
     raise _import_error from e
 import numpy as np
+import numpy.typing as nptyping
 
-# All rc params are available here
-# https://matplotlib.org/3.2.1/tutorials/introductory/customizing.html#customizing-with-matplotlibrc-files
-rc_params       = { "font.size": 26, "legend.fontsize": 24, "legend.framealpha": 1, "legend.edgecolor": (0, 0, 0, 1) }
-rc_params_small = { **rc_params, "font.size": 22, "legend.fontsize": 20 }  # Same but with smaller font
 
-def update_rc_params(rc_params: dict[str, Any]):
-    """ Update matplotlib parameters - utility function for preventing always having to look it up """
-    plt.rcParams.update(rc_params)
-
-# Colours
-base_colours: tuple[str] = tuple(mcolour.BASE_COLORS)  # 8 colours
-tab_colours:  tuple[str] = tuple(mcolour.TABLEAU_COLORS)  # 10 colours
-colours:      tuple[str] = tab_colours[:-2] + base_colours[:-1]  # 15 unique matplotlib colours
-
-# Common figure sizes
-figsize_std  = (15, 10)
```

Comment:
Vi vil ikke beholde de her for bagvendt kompatabilitet?

Reply:
Den her opdatering er en krigserklæring mod bagudkompatibilitet (dog den sidste), medmindre vi en dag laver 2.0.0, så det ser jeg ingen grund til